loc/tests/test_schema.py

```diff
@@ -332,3 +334,69 @@ def test_email_letter_works(db, graphql_client, mailoutbox):
     )['data']['emailLetter']
     assert result == {'errors': [], 'recipients': ['boop@jones.com']}
     assert len(mailoutbox) == 1
+
+
+RA_DATA_QUERY = '''
+query {
+    session {
+        onboardingInfo {
+            hasCalled311
+        }
+    }
+}
+'''
+
+RA_MUTATION = """
+mutation {
+    reliefAttempts(input: {}) {
```

Comment:
Hmm I wonder if this is the cause of the error you're getting?  It seems like you're hard-coding the input to be empty... you'll actually want to do a funky thing like this, from the HP action schema tests:

```graphql
        mutation MyMutation($input: TenantChildrenInput!) {
            output: tenantChildren(input: $input) {
                errors { field, messages }
            }
        }
```

There the input is defined to be a thing that's passed in from your code, rather than hard-coded into the query or whatever.  (I should really know the terminology here but I don't, sorry!)

Reply:
Ok cool, thank you!